config.example.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
   symbol: 'SYM',
   // Project description
   description: 'An awesome NFT series',
+  // Path to image to use as the main image for the collection
+  image: 'images/1.png',
```

Comment:
I was under the impression we'd use the image from the contract-level metadata. Typically this is a GIF with rotating images from the collection.

Reply:
We're going to have to document how to make this.

Reply:
There is no reason they couldn't set this image to a GIF with rotating images from the collection.